crates/bevy_time/src/time.rs

```diff
@@ -188,7 +190,7 @@ use bevy_utils::Duration;
 /// ```
 #[derive(Resource, Debug, Copy, Clone)]
 #[cfg_attr(feature = "bevy_reflect", derive(Reflect), reflect(Resource, Default))]
-pub struct Time<T: Default = ()> {
+pub struct Time<T: Domain = ()> {
```

Comment:
_Technically_ a breaking change, but I have no idea who needed a non-`Send`/`Sync`/`'static` `Time`.

Reply:
Yea, `Resource` already puts a requirement on it for `Send`/`Sync`/`'static`, so not all that much of a change.